index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <main id="id-main-container">
+      <header><!--header of the document-->
+        <h1 id='title'> “Football Survey Form”</h1>
+        <p id='description'>Complete your registration</p>
+      </header>
+
+      <form id="survey-form">
+
+        <div id="id-getUserInfos"><!--gets the user infos and let him chose one player from the dropdown-->
+          <!--fieldset to enter information-->
+          <label id="name-label">Name</label>
```

Comment:
Kindly, it would be better if you use class attribute instead of id for all parts of your code. The id attribute can not be repeated in a code. Also, should be used only for the big parts of your code. You are the best!